packages/odyssey/CHANGELOG.md

```diff
@@ -10,25 +10,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rebrand type and color changes
 - bugfix/ee-463 - fixes table cell overflow by wrapping long strings
```

Comment:
We can remove this line now that it's being released

Reply:
good catch, I had cut this branch off of master. fixed.